lesson_01/lesson_01_hw.html

```diff
@@ -0,0 +1,193 @@
+        position: absolute;
+        bottom: 84px;
+        width: 261px;
+        height: 280px;
+        display: none;
+      }
+      main .featuredFlexContainer .featuredFlexChild .featuredProductHover button {
+        width: 123px;
+        height: 39px;
+        margin: 94px 70px 144px;
+        background: rgba(0, 0, 0, 0) url('../img/cart2.png') no-repeat 5px;
+        border: 1px solid #ffffff;
+        color: #ffffff;
+        font-size: 13px;
+        font-weight: 700;
+        line-height: 39px;
+        cursor: pointer;
+      }
+      main .featuredFlexContainer .featuredFlexChild .featuredProductHover button span {
+        padding-left: 20px;
+      }
+      main .featuredFlexContainer .featuredFlexChild:hover {
+        box-shadow: 0 5px 8px rgba(0, 0, 0, 0.16);
+      }
+      main .featuredFlexContainer .featuredFlexChild:hover .featuredProductHover {
+        display: block;
+      }
+      main .featuredFlexContainer .featuredFlexChild:hover .featuredProductText h6 {
+        display: block;
+      }
+      main .featuredFlexContainer .featuredFlexChild .featuredProductText {
+        margin: 20px 0 17px 14px;
+      }
+      main .featuredFlexContainer .featuredFlexChild .featuredProductText .featuredProductName {
+        color: #222;
+        font-size: 13px;
+        font-weight: 400;
+        margin-bottom: 17px;
+      }
+      main .featuredFlexContainer .featuredFlexChild .featuredProductText .featuredProductPrice {
+        color: #f16d7f;
+        font-size: 16px;
+        font-weight: 700;
+        display: inline-block;
+      }
+      main .featuredFlexContainer .featuredFlexChild .featuredProductText h6 {
+        float: right;
+        color: #e4af48;
+        font-size: 15px;
+        font-weight: 400;
+        display: none;
+        margin-right: 10px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <ul class="menu">
+          <li><a href="#">Main page</a></li>
+          <li><a href="#">Catalog</a></li>
+          <li><a href="#">Cart</a></li>
+          <li><a href="#">Sign in</a></li>
+          <li><a href="#">Sign up</a></li>
+        </ul>
+      </nav>
+      <button class="cart-button" type="button" name="button">Cart</button>
+    </header>
+
+    <main>
+      <div class="featuredFlexContainer"></div>
+    </main>
+
+    <script type="text/javascript">
+      const goods = [
+        { title: 'Shirt', price: 150 },
+        { title: 'Socks', price: 50 },
+        { title: 'Jacket', price: 350 },
+        { title: 'Shoes', price: 250 },
+      ];
+
+      /*const renderGoodsItem = ({ title, price }) => {
+        return `
+                <div class="goods-item">
+                  <h3>${title}</h3>
+                  <p>${price} $</p>
+                  <button class="add-button">Add to cart</button>
+                </div>
+               `;
+      };*/
+
+      const renderGoodsItem = ({ title, price }, i) => {
```

Comment:
А где не получилось значение по умолчанию из-за деструктуризации добавить? Его тут можно по-всякому добавлять. Пример:
```suggestion
      const renderGoodsItem = ({ title, price = 50 } = { title: 'Носки' }, i) => {
```